Orange/widgets/visualize/owscatterplot.py

```diff
@@ -578,7 +579,9 @@ def run(self):
                     y = y_full[valid]
                     knn = NearestNeighbors(n_neighbors=self.k).fit(X)
                     ind = knn.kneighbors(return_distance=False)
-                    score = norm * np.sum(y[ind] == y.reshape(-1, 1))
+                    score = norm * np.sum(y[ind] == y.reshape(-1, 1)) \
```

Comment:
@markotoplak Any chance of turning this into an ordinary if? I find these expressions difficult to read, and when they stretch across three lines, it's just a matter of swapping them around...